### DISCUSSION TITLE
LangManagement - where are values stored?

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15784] [How to] [answered]
Added custom language and in LangManagement translated fields/picklists. Where system storing this language? Cannot find in database

**Comment by JohnMaverick** [comment]
Good question, was looking for that too. By the way, how to call system language via API?

**Reply by mariuszkrzaczkowski** [reply]
what do you mean?

**Reply by JohnMaverick** [reply]
API pulling picklist values. As per default is English, how to pull other translated picklist... a bit stuck on it

**Reply by rskrzypczak** [reply]
Set in configuration panel:
![image](https://user-images.githubusercontent.com/10221999/157626404-fa5b018e-f8e4-4609-8019-b35f18055b89.png)
or add to header in request: _accept-language_

**Reply by mariuszkrzaczkowski** [reply]
another way is to set the language when logging in to the API

```
{
    "userName": "partner@yetiforce.com",
    "password": "partner",
    "code": "",
    "params": { 
        "language" : "en-US"
    }
}
```
https://doc.yetiforce.com/api/#/Users/1ac849ff510b6b65243b46ffb9e48a9b
